less/selectlist.less

```diff
@@ -13,15 +17,14 @@
 		min-width: inherit;
 	}
 	.btn.dropdown-toggle {
-		white-space: normal;
+		white-space: nowrap;
```

Comment:
This is already set on .btn.  Might as well just remove this line completely if you're setting it back to nowrap.